Dockerfile

```diff
@@ -82,9 +81,13 @@ RUN curl -o /tmp/git-2.7.0.tar.gz https://www.kernel.org/pub/software/scm/git/gi
 RUN mkdir -p /etc/ld.so.conf.d/ && echo "/opt/curl/lib" > /etc/ld.so.conf.d/optcurl.conf && ldconfig
 
 RUN /bin/bash -l -c "CPPFLAGS='-I/usr/local/rvm/gems/ruby-2.2.2/include' rvm install 2.2.2 --with-openssl-dir=/opt/openssl" && \
-    /bin/bash -l -c "rvm --default use 2.2.2" && \
-    /bin/bash -l -c "gem install bundler --no-ri --no-rdoc" && \
-    rm -rf /usr/local/rvm/src/ruby-2.2.2
+    /bin/bash -l -c "rvm --default use 2.2.2"
+
+# bootstap our CERTS
+RUN /opt/curl/bin/curl -kfsSL curl.haxx.se/ca/cacert.pem \
+                       -o $(/bin/bash -l -c "ruby -ropenssl -e 'puts OpenSSL::X509::DEFAULT_CERT_FILE'")
+
+RUN /bin/bash -l -c "gem install bundler --no-ri --no-rdoc" && rm -rf /usr/local/rvm/src/ruby-2.2.2
```

Comment:
Could we do these 3 `RUN`s in just one `RUN`? The image has grown in size from 1.5GB to 2GB and I think it's because of this.